### PR TITLE
maclaunch 0.2 (new formula)

### PR DESCRIPTION
**Pull request by hazcod**
I wrote a little bash script that allows you to list, enable and disable any launch agents/daemons.

### CODE REVIEW

File: Formula/maclaunch.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class Maclaunch < Formula
+  desc "Command-line utility for managing your launch agents and daemons."
+  homepage "https://github.com/HazCod/maclaunch"
+  url "https://github.com/HazCod/maclaunch/archive/0.2.tar.gz"
+  sha256 "1a2c7f62cb5331df9804c481fecc41e29b4dff29"
+
+  def install
+    # for clarity, remove .sh
+    mv "maclaunch.sh", "maclaunch"
+    bin.install "maclaunch"
```
Comment (bfontaine) [inline Formula/maclaunch.rb]: Use `bin.install "maclaunch.sh" => "maclaunch"`.

File: Formula/maclaunch.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  end
+
+  test do
+    # just run the command to test the install
```
Comment (bfontaine) [inline Formula/maclaunch.rb]: Please remove this comment.

**Review by bfontaine — CHANGES_REQUESTED** [review]

**Comment by bfontaine** [comment]
Given that it’s a self-submission of a new project we’re going to pass on this for now. You can maintain the formula in your own tap and re-submit it here once it’s popular. Sorry!

https://github.com/Homebrew/brew/blob/master/docs/How-to-Create-and-Maintain-a-Tap.md

**Comment by HazCod** [comment]
@bfontaine ; just a question, how many repo stars do you require typically? 
Since I mentioned this on reddit, perception was positive.

**Comment by MikeMcQuaid** [comment]
@HazCod It's not criticism of your project but just that you should wait for someone from your community to submit this when it's more established.

**Pull request closed without merging**